R/01-createDuplicateModal.R

```diff
@@ -0,0 +1,131 @@
+                You can then select the degree of similarity, independently for each column, which specifies how similar cases must be to be classified as duplicates.<br><br>
+                For numeric columns, you can choose between 'Exact Match' and 'Rounded Match'.
+                For the 'Exact Match' selection, observations are only classified as duplicated if the values match exactly.
+                For 'Rounded Match', values are classified as identical if they are identical after rounding.
+                For this reason, you can specify a rounding factor if you have selected 'Rounded Match'.
+                A value of 2, for example, means that the values are rounded to 2 decimal places before being checked for equality.
+                A value of 0 means that the values are rounded to whole numbers.
+                You can also enter negative numbers, which will result in rounding to the power of ten, e.g. the value -2 rounds to the nearest hundred.<br>
+                For text columns you can choose between 'Case Sensitive', where a distinction is made between upper and lower case letters and
+                'Case Insensitive', where no distinction is made.<br><br>
+                You then have the option to display the duplicates and/or remove them from the dataset."),
+      hr(),
+      selectizeInput(
+        inputId = ns("variables"),
+        label = "Duplicate Identifikation Columns",
+        choices = vars,
+        multiple = TRUE
+      ),
+      hr(),
+      fluidRow(
+        column(
+          4,
+          selectizeInput(
+            inputId = ns("selectedVariable"),
+            label = "Degree of Similarity for Variable...",
+            choices = NULL
+          )
+        )
+      ),
+      fluidRow(
+        column(
+          4,
+          shinyjs::hidden(
+            selectizeInput(
+              inputId = ns("textSimilarity"),
+              label = "Text Similarity",
+              choices = c(
+                "Case Sensitive",
+                "Case Insensitive"
+              ),
+              selected = "Case Sensitive"
+            )
+          )
+        )
+      ),
+      fluidRow(
+        column(
+          4,
+          shinyjs::hidden(
+            selectizeInput(
+              inputId = ns("numericSimilarity"),
+              label = "Numeric Similarity",
+              choices = c(
+                "Exact Match",
+                "Rounded Match"
+              ),
+              selected = "Exact Match"
+            )
+          )
+        ),
+        column(
+          4,
+          conditionalPanel(
+            condition = "input.numericSimilarity == 'Rounded Match'",
+            ns = ns,
+            numericInput(
+              inputId = ns("rounding"),
+              label = "Rounding Factor",
+              value = 0,
+              min = -10,
+              max = 10
+            )
+          )
+        )
+      ),
+      checkboxInput(
```

Comment:
What about shifting the inputs into one row? Than the user would need less scrolling...

![image](https://user-images.githubusercontent.com/16759098/215456224-0e38dced-ea8a-4c07-ad59-4a0ac68b0757.png)